From cb87c89745c93969c3742e7a8a11d546c00107c8 Mon Sep 17 00:00:00 2001
From: Juan Pablo Ugarte <juanpablougarte@gmail.com>
Date: Fri, 11 Aug 2017 18:17:18 -0300
Subject: GladeSignalModel: rename g_ptr_array_find() to prt_array_find()

GLib 2.54 introduced g_ptr_array_find() which clashes with our own function
---
 gladeui/glade-signal-model.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gladeui/glade-signal-model.c b/gladeui/glade-signal-model.c
index 2b383e5..aed27b1 100644
--- gladeui/glade-signal-model.c
+++ gladeui/glade-signal-model.c
@@ -65,7 +65,7 @@ G_DEFINE_TYPE_WITH_CODE (GladeSignalModel, glade_signal_model, G_TYPE_OBJECT,
                                                 gtk_tree_drag_source_iface_init))
 
 static gint
-g_ptr_array_find (GPtrArray *array, gpointer data)
+ptr_array_find (GPtrArray *array, gpointer data)
 {
   gint i;
   for (i = 0; i < array->len; i++)
@@ -524,7 +524,7 @@ glade_signal_model_get_path (GtkTreeModel *model, GtkTreeIter *iter)
 	    {
 	      if (handlers)
 		{
-		  gint handler_index = g_ptr_array_find (handlers, handler);
+		  gint handler_index = ptr_array_find (handlers, handler);
 		  if (handler_index == -1) /* dummy handler */
 		    {
 		      index1 += handlers->len;
@@ -582,7 +582,7 @@ glade_signal_model_get_value (GtkTreeModel *model,
         {
           GPtrArray *handlers = g_hash_table_lookup (sig_model->priv->signals,
                                                      glade_signal_get_name (signal));
-          if (!handlers || !handlers->len || g_ptr_array_find (handlers, signal) == 0)
+          if (!handlers || !handlers->len || ptr_array_find (handlers, signal) == 0)
             g_value_set_boolean (value, TRUE);
           else
             g_value_set_boolean (value, FALSE);
@@ -718,7 +718,7 @@ glade_signal_model_iter_next (GtkTreeModel *model, GtkTreeIter *iter)
 	}
       else if (handlers)
 	{
-	  gint new_index = g_ptr_array_find (handlers, handler) + 1;
+	  gint new_index = ptr_array_find (handlers, handler) + 1;
 	  if (new_index < handlers->len)
 	    {
 	      glade_signal_model_create_signal_iter (sig_model, widget,
-- 
cgit v0.12
